phoenix_project/pento/lib/pento_web/live/wrong_live.ex

```diff
@@ -0,0 +1,88 @@
+defmodule PentoWeb.WrongLive do
+    use PentoWeb, :live_view
+
+    def mount(_params, _session, socket) do
+        {:ok, assign(socket, score: 0, goal: Enum.random(1..10), objective: :false,  message: "Make a guess:")}
+    end
+
+    def render(assigns) when assigns.objective == :true do
```

Comment:
Ovdje bi bilo bolje da nemas dvije render() funkcije, jer su obje vrlo slicne pa je vecina koda duplicirana. Probaj rijestiti to s jednom funkcijom, a onda unutar nje provjeris da li je assigns.objective jednako true, pa na temelju toga napravis razliku u renderiranju.

Znaci imat ces nesto ovako:

```elixir
<%= if @objective do %>
  # ovdje ide kod ako je objective true
<% else %>
  # ovdje ide kod ako @objective nije true
<% end %>
```